docs/getting-started-guides/openstack-heat.md

```diff
@@ -28,6 +28,7 @@ If you already have the required versions of the OpenStack CLI tools installed a
 
 ```sh
 sudo pip install -U --force 'python-openstackclient==2.4.0'
+sudo pip install -U --force 'python-heatclient==1.1.0'
```

Comment:
Yes, the openstackclient won't do heat things without the heatclient. I only caught it after doing a cleanroom test.